assignments/objects.js

```diff
@@ -19,20 +19,34 @@ const example = {
 
 // Write your intern objects here:
 
+ function constructInter (i, n, e, g) {
+  const newObj = {
+    id: i,
+    name: n,
+    email: e,
+    gender: g,
+  }
+  return newObj;
+}
 
+const mitzi = constructInter(1, "Mitzi", "mmelloy0@psu.edu","F"); 
+const kennan = constructInter(2, "kdiben1@tinypic.com", "Kennan","M");
+const kevan = constructInter(3, "kmummery2@wikimedia.org", "Kevan","M");
+const gannie = constructInter(4, "gmartinson3@illinois.edu", "Gannie","M");
+const antonietta = constructInter(5, "adaine5@samsung.com", "Antonietta","F");
```

Comment:
Never seen it written like this in one line. As nice as this is having it all in one line, it will be easier to read if you have it spaced out where each key and value have their own line. Either way. Great work with this.